ark/attest/assert/chainableAssertions.ts

```diff
@@ -229,6 +233,7 @@ export type ChainContext = {
 
 export type functionAssertions<kind extends AssertionKind> = {
 	throws: inferredAssertions<[message: string | RegExp], kind, string>
+	completions: inferredAssertions<[Completions], kind, Completions>
```

Comment:
The `comparableValueAssertions` are always available, so you don't need this here as well. Maybe rename `comparableValueAssertions` to `baseValueAssertions` to make that clearer (they used to not be available on functions).